package.json

```diff
@@ -67,7 +67,7 @@
     "cache-loader": "^3.0.1",
     "clean-webpack-plugin": "^2.0.1",
     "core-js": "^2.5.3",
-    "css-loader": "^3.1.0",
+    "css-loader": "^3.2.0",
```

Comment:
Why these changes? Should do them in a separate PR.

Reply:
I didn't mess with the package.json and I am not sure as to what to do.

Reply:
You should be able to just checkout fresh copies from develop again (after merging with HEAD).